docs/antrea-network-policy.md

```diff
@@ -547,12 +577,14 @@ particular Namespaces.
 
 **namespaces**: A `namespaces` field allows users to perform advanced matching on
 Namespace objects which cannot be done via label selectors. Currently, the
-`namespaces` field has only one matching strategy, `Self`. If set to `Self`, it indicates
-that the corresponding `podSelector` (or all Pods if `podSelector` is not set)
+`namespaces` field has two matching strategies, `Self` and `NotSelf`. If set to `Self`,
+it indicates that the corresponding `podSelector` (or all Pods if `podSelector` is not set)
```

Comment:
How about "it indicates that for a given target Pod (selected by either the policy-level AppliedTo or the rule-level Applied-To) to apply the rule, the rule's `podSelector` should only select Pods in the same Namespace as the target Pod"?

Reply:
Sounds good, keeping "either through a policy-level AppliedTo or a rule-level Applied-To" also helps understand how to use `namespaces`.